### DISCUSSION TITLE
Browser push notifications

### DISCUSSION THREAD

**Discussion by pkoepke** [NickColley/semaphore#73] [New features and ideas]
It would be nice to (optionally) get push notifications from instances.

Sorry if this has been suggested before, I searched past issues and didn't find anything. I did find this issue from Pinafore https://github.com/nolanlawson/pinafore/issues/19 which indicates push support was added to Pinafore, but I can't find anything related to push notifications in Semaphore.

**Comment by easrng** [comment]
it's in your instance settings as "Push Notifications"

**Comment by pkoepke** [comment]
Thanks, the problem was I didn't realize I had to tap on the instance name under settings > instances to find the push notification settings.
